math.asm

```diff
@@ -163,3 +164,64 @@ end:    INX
     sta MSB+1,x
     inx
     rts
+
+    +BACKLINK "invert", 6
+INVERT
+    lda MSB, x
+    eor #$ff
+    sta MSB, x
+    lda LSB, x
+    eor #$ff
+    sta LSB,x
+    rts
+
+    +BACKLINK "negate", 6
+NEGATE
+    jsr INVERT
+    jmp ONEPLUS
+
+    +BACKLINK "abs", 3
+ABS
+    lda MSB,x
+    bmi NEGATE
+    rts
+
+DABS_STAR
+    lda MSB,x
+    eor MSB+1,x
+    pha
+    jsr ABS
+    inx
+    jsr ABS
+    dex
+    jsr U_M_STAR
+    pla
+    rts
+
+    +BACKLINK "*", 1
+    jsr DABS_STAR
+    inx
+    and #$ff
```

Comment:
it would be a net saving to replace `and #$ff` with the `pla` from DABS_STAR. (`m*` would need to update, too)

Reply:
If that is done, `jsr U_M_STAR , rts` can also be collapsed to `jmp U_M_STAR`.

Reply:
This was the original design, but causes a CPU JAM when compiling `labels`.  I didn't look into it too deeply.  There's only 2 multiplies in there so ???

Reply:
ah, of course it doesn't work, because the return adress needs to stay on top of return stack.